client/src/constants/Header.ts

```diff
@@ -9,4 +9,9 @@ export const headerLinks: IHeaderLink[] = [
   { name: 'chapters', label: 'Chapters', href: '/chapters' },
   { name: 'login', label: 'Login', href: '/login' },
   { name: 'register', label: 'Register', href: '/register' },
+  {
+    name: 'google',
+    label: 'google',
+    href: 'http://localhost:5000/auth/google/',
+  },
```

Comment:
why does google have its own link in the header? shouldn't this just be part of logging in?

Reply:
This comment still stands^